packages/patternfly-4/src/content/design-guidelines/usage-and-behavior/forms/design.md

```diff
@@ -41,78 +42,67 @@ The PatternFly components listed in the following sections can be used in a numb
 [Form](/documentation/react/components/form), [Button](/documentation/react/components/button), [Checkbox](documentation/react/components/checkbox), [Dropdown](/documentation/react/components/dropdown), [Progress](/documentation/react/components/progress), [Radio](/documentation/react/components/radio), [Select](/documentation/react/components/select), [Switch](/documentation/react/components/switch), [TextArea](/documentation/react/components/textarea), [Text](/documentation/react/components/text), [Popover](/documentation/react/components/popover), [TextInput](/documentation/react/components/textinput), [Tooltip](/documentation/react/components/tooltip)
 
 ## Usage
-### Use progressive disclosure
+There's a lot to think about when you're building a form, and there's no one-size-fits-all solution. These usage guidelines are here to help you use the right combination of elements and content to gather the information you need from your users.
 
-![Progressive disclosure](img/progressive-disclosure.png)
-*Progressive disclosure example*
-
-Use progressive disclosure to hide and show fields based on the users’ selection. This approach keeps the workflow short and allows the user focus on only the fields relevant to them. It also avoids using color to indicate whether or not a field is available, making it suitable for colorblind users.
+### Provide the right data input
+PatternFly provides a variety of components that enable a user to make a simple selection. Each component was created to serve a specific use case. For example, you might need to allow your user to select one or more items from a list, but should you use a checkbox or a multi-select dropdown? It depends on the number of items your user needs to choose from.    
 
-### Choose the right data input
-There are multiple components that enable a user to make a simple selection. To learn more about data input, visit the [text and data input design guidelines](/design-guidelines/usage-and-behavior/data-input).
+Learn more about data input in the [text and data input design guidelines](/design-guidelines/usage-and-behavior/data-input).
 
 ### Provide the right kind of help
-There are multiple ways to provide help on forms.
+There are a few different ways to provide help on forms. The type of help you choose will depend on the content you need to relay to your user.  
 
 **Field level help**
```

Comment:
I'm not sure this approach is still valid. I know we had multiple discussions about accessibility of putting the Help icon to the right of the field. @jgiardino @mceledonia can you remind me of where we ended up on this?

Reply:
The recommendation from deque is that if a form field includes a button that provides help for that form, then the button should precede the input field in the DOM, and the label for the button should clearly associate it with the input (e.g. repeat the input label in the button label). The reason for this is so that screen reader users know that help is available for an input before they get to the input.